pyproject.toml

```diff
@@ -49,6 +49,7 @@ openai = [
 
 [project.scripts]
 ddtrace-run = "ddtrace.commands.ddtrace_run:main"
+crashtracker_exe = "ddtrace.commands.crashtracker_exe:main"
```

Comment:
is this something we want to expose to customers? any reason to not just invoke directly from the code like `os.path.join(project_root, 'internal', 'crashtracker_exe')` ?

Reply:
doing that means we also don't have to worry about `PATH` at all.